src/OpenTK/Platform/X11/XI2MouseKeyboard.cs

```diff
@@ -397,6 +399,14 @@ private void UpdateDevices()
                     }
                     XI.FreeDeviceInfo((IntPtr)list);
                 }
+
+                // Resture current mouse position
```

Comment:
Might be a slight typo, I think it should be `Restore` and not `Resture`.

Reply:
yes, a typo